app/views/content/subjects/engineers-teach-physics/_article.html.erb

```diff
@@ -64,7 +64,7 @@
      <p>You'll also be invited to an annual community day and have the opportunity to network with existing physics teachers.
      </p>
 
-    <p>Courses starting in the 2022/23 academic year are now closed. You’ll be able to find courses starting in the 2023/24 academic year in autumn 2022.
+    <p>You can <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">find teacher training courses</a> and filter by 'physics' then 'engineers teach physics'.
```

Comment:
```suggestion
    <p>You can <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">find teacher training courses</a> and filter by 'physics' then 'engineers teach physics'.
```
```suggestion
    <p>You can <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">search for Physics teacher training courses</a> and then filter by 'Engineers teach physics'.
```
Is this a bit easier? But happy for you to disagree @EllieNodder 

I've capitalised 'Engineers' as that is how it is written on Find - again, please disagree if necessary!

![image](https://user-images.githubusercontent.com/87642394/199517991-0f2f3f62-96fd-4a39-a81c-3da44bfef8c7.png)

Reply:
just didn't know if we should put 'physics' in the link text, just because we're not linking them to **physics** courses but all courses - but I guess we're saying 'search for', so maybe it's okay?

Reply:
Yes, I was a bit in two minds about that - which is why I thought search might be better than find in this case. Happy to go with whichever you think best!